app/src/sharedTest/java/org/oppia/android/app/devoptions/DeveloperOptionsActivityTest.kt

```diff
@@ -241,6 +249,48 @@ class DeveloperOptionsActivityTest {
     }
   }
 
+  @Test
+  fun testDeveloperOptionsFragment_clickMarkChaptersCompleted_opensMarkChaptersCompletedActivity() {
+    launch<DeveloperOptionsActivity>(
+      createDeveloperOptionsActivityIntent(
+        internalProfileId = internalProfileId
```

Comment:
we can skip such named arguments where the named argument is the same as the argument.

Reply:
removed at all locations.